src/data/types/labeltype.cpp

```diff
@@ -41,9 +41,9 @@ QString LabelType::label() const
     return QString();
 }
 
-QString LabelType::baseIconFile() const
+Utils::FilePath LabelType::baseIconFile() const
 {
-    return QString();
+    return Utils::FilePath();
```

Comment:
```suggestion
    return {};
```
to avoid changes in the future :)